spec/system/claims/schools/users/copy_view_users_spec.rb

```diff
@@ -1,68 +1,68 @@
 require "rails_helper"
 
 RSpec.describe "Invite and view a users details copy", service: :claims, type: :system do
-  scenario "I sign in as a lead mentor user and invite a user to a school and view that users details" do
+  scenario do
     given_a_school_exists_with_claims_users
-    and_i_am_signed_in_as_a_lead_mentor_user
+    and_i_am_signed_in_as_a_user
```

Comment:
```suggestion
  and_i_am_signed_in
```
Given that only users can sign in I think we can shorten this?